bubbles/main.py

```diff
@@ -0,0 +1,192 @@
+import logging
+import os
+import pathlib
+import sys
+
+import click
+from click.core import Context
+from slack_bolt.adapter.socket_mode import SocketModeHandler
+
+from bubbles import __version__
+from bubbles.config import (
+    app,
+    COMMAND_PREFIXES,
+    DEFAULT_CHANNEL,
+)
+from bubbles.interactive import InteractiveSession
+from bubbles.plugins import PluginManager as PM
+from bubbles.reaction_added import reaction_added_callback
+from bubbles.tl_commands import enable_tl_jobs
+from bubbles.tl_utils import tl
+
+PluginManager: PM
```

Comment:
I'm not a fan of this, why does this variable have the same name as its class?
Can we rename that to `plugin_manager` or something similar?

Reply:
Good catch. Was mostly focused on backwards compatibility but I guess we don't actually need that. Time to change!